generator/src/test/java/com/scottlogic/deg/generator/cucumber/LessThan.feature

```diff
@@ -2,12 +2,332 @@ Feature: User can specify that a numeric value is lower than, but not equal to,
+     Given foo is anything but less than 4
+       And foo is granular to 1
+       And the generator can generate at most 5 rows
+     Then the following data should be generated:
+       | foo |
+       | 4   |
+       | 5   |
+       | 6   |
+       | 7   |
+       | 8   |
+
+#unsure on what the output should be here
+Scenario: not lessThan run against a non contradicting not granularTo should be successful (not lessThan 5 AND not granularTo 1)
+     Given foo is anything but less than 5
+       And foo is anything but granular to 1
+       And the generator can generate at most 5 rows
+     Then some data should be generated
+
+#after
+@ignore #589: produces big-integer numbers rather than integer values, not sure why
+Scenario: lessThan run against a non contradicting after should be successful (lessThan 1 AND after 2019-01-01T00:00:00.00)
+     Given foo is less than 1
+       And foo is after 2019-01-01T00:00:00.000
+       And the generator can generate at most 5 rows
+     Then the following data should be generated:
+       | foo         |
+       | -2147482624 |
+       | -2147482623 |
+       | -2147482622 |
+       | -2147482621 |
+       | -2147482620 |
+
+@ignore #589: produces big-integer numbers rather than integer values, not sure why
+Scenario: lessThan run against a non contradicting not after should be successful (lessThan 1 AND not after 2019-01-01T00:00:00.00)
+     Given foo is less than 1
+       And foo is anything but after 2019-01-01T00:00:00.000
+       And the generator can generate at most 5 rows
+     Then the following data should be generated:
+       | foo         |
+       | -2147482624 |
+       | -2147482623 |
+       | -2147482622 |
+       | -2147482621 |
+       | -2147482620 |
+
+#afterOrAt
+@ignore #589: produces big-integer numbers rather than integer values, not sure why
+Scenario: lessThan run against a non contradicting after should be successful (lessThan 1 AND afterOrAt 2019-01-01T00:00:00.00)
+     Given foo is less than 1
+       And foo is after or at 2019-01-01T00:00:00.000
+       And the generator can generate at most 5 rows
+     Then the following data should be generated:
+       | foo         |
+       | -2147482624 |
+       | -2147482623 |
+       | -2147482622 |
+       | -2147482621 |
+       | -2147482620 |
+
+@ignore #589: produces big-integer numbers rather than integer values, not sure why
+Scenario: lessThan run against a non contradicting not after should be successful (lessThan 1 AND not afterOrAt 2019-01-01T00:00:00.00)
+     Given foo is less than 1
+       And foo is anything but after or at 2019-01-01T00:00:00.000
+       And the generator can generate at most 5 rows
+     Then the following data should be generated:
+       | foo         |
+       | -2147482624 |
+       | -2147482623 |
+       | -2147482622 |
+       | -2147482621 |
+       | -2147482620 |
```

Comment:
same as above, not sure on these expected results

Reply:
Ordering again, they would be I think but would be last

Reply:
Ref #594